server/src/services/auth.service.ts

```diff
@@ -226,7 +226,7 @@ export class AuthService extends BaseService {
       const storageQuota = this.getClaim(profile, {
         key: storageQuotaClaim,
         default: defaultStorageQuota,
-        isValid: (value: unknown) => isNumber(value) && value >= 0,
+        isValid: (value: unknown) => isNumber(value) && Number(value) >= 0,
```

Comment:
It looks like the number conversion won't do anything, because `isNumber` checks if the value is a number and returns false otherwise. Shouldn't the value be converted to a number beforehand?

Reply:
Oh you're right, good catch!

Reply:
Actually I don't even need the `isNumber` check because `Number.NaN >= 0 === false`